### PR TITLE
doc: added credentials description

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #1031 
Added detailed description for credentials.

### CODE REVIEW

File: .wiki/Configuration-Credentials.md [hunk 1]
```diff
@@ -24,63 +42,35 @@ credentials:
...
-`ARTIPIE_USER_NAME` and `ARTIPIE_USER_PASS`. For example, you start
-Docker container with the `-e` option:
+### Credentials type `github`
 
-```bash
-docker run -d -v /var/artipie:/var/artipie` -p 80:80 \
-  -e ARTIPIE_USER_NAME=artipie -e ARTIPIE_USER_PASS=qwerty \
-  artipie/artipie:latest
-```
+If the `type` is set to `github`, GitHub username with `github.com/` prefix and [personal access
```
Comment (dgarus) [inline .wiki/Configuration-Credentials.md:58]: It would be clearer to give additionally an example:
```
github.com/{username}
```

**Review by dgarus — CHANGES_REQUESTED** [review]
@olenagerasimova thanks!
There is a small comment.

**Comment by olenagerasimova** [comment]
@dgarus thanks, added

**Review by dgarus — APPROVED** [review]
@olenagerasimova thanks!

**Pull request merged**